connectors/python/.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+source = symbolconnectors
```

Comment:
can you remind me what is this? (I mean why source == `symbolconnectors`)

Reply:
it configures code coverage to run on all py files in symbolconnectors directory